sphinx/source/docs/user_guide/tools.rst

```diff
@@ -529,7 +524,7 @@ sets ``mode = "vline"``.
 
 .. _userguide_tools_formatting_tooltip_fields:
 
-Formatting Tooltip Fields
+Formatting tooltip fields
 '''''''''''''''''''''''''
 
 By default, values for fields (e.g. ``@foo``) are displayed in a basic numeric
```

Comment:
Same thing here - to make reading as easy as possible, I'd recommend using English terms instead of Latin ones (https://developers.google.com/style/abbreviations#dont-use). So:

> values for fields (``@foo``, for example) are displayed in ...